styles/styles.scss

```diff
@@ -69,34 +176,38 @@ $floatingShadow: 0px 2px 4px rgba(0, 0, 0, 0.13);
   justify-content: center;
   align-items: center;
   height: 50px;
-  padding: $base $base * 3;
-
+  padding: 14px 24px;
+  box-sizing: border-box;
+  margin: $base;
   font-family: $fonts;
   font-size: 16px;
   font-weight: 500;
   line-height: 24px;
-
-  border-radius: 25px;
+  text-align: center;
+  align-items: center;
+  min-width: 146px;
+  height: 48px;
+  top: 24px;
```

Comment:
not sure `top` is needed. also we need a translate property so the buttons look like they're being pressed in
see https://codepen.io/anthonycr/pen/GRgExEO for an example